build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
```

Comment:
update readme that instead of using `yarn build` use this  to create build artifacts (before copying / use with nginx)